indico/modules/events/abstracts/views.py

```diff
@@ -46,6 +46,7 @@ def getCSSFiles(self):
                 self._asset_env['markdown_sass'].urls() +
                 self._asset_env['selectize_css'].urls() +
                 self._asset_env['abstracts_sass'].urls() +
+                self._asset_env['reviewing_sass'].urls() +
```

Comment:
I'd include that before abstracts_sass since - if you ever have to - overriding would happen the other way round (abtracts is more specific than reviewing).